backend.rb

```diff
@@ -35,51 +36,51 @@ class Backend < Sinatra::Base
     # Make key-value pairs from the JSON body available in the 'params' hash
     params.merge!(JSON.parse(request.body.read))
 
-    basic_auth_options = {
-      basic_auth: { username: API_KEY, password: API_SECRET}
-    }
-
     logging_options = {
```

Comment:
@grozen
I think it's worth mentioning in the beginning of our read.me, that we log all our http server request to the log, in a curl format. 
So there is no real need to understand the code, just run it.